### PR TITLE
feat(AI Analytics): 19963 Update AI analytics placeholder

### PR DESCRIPTION
**Pull request by albaranau**
https://kontur.fibery.io/Tasks/Task/Update-AI-Analytics-placeholder-19963

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `LLMAnalyticsPlaceholder` component that provides user guidance related to LLM analytics, including clickable links for navigation.
	- Enhanced localization with updated error messages and a new structured guide for AI insights.

- **Bug Fixes**
	- Corrected a typographical error in the error message for advanced analytics.

- **Style**
	- Added new CSS styles for the `LLMAnalyticsPlaceholder` component to improve layout and interactivity.
	- Adjusted margin styling in the `LLMAnalyticsEmptyState` component for better spacing.

### CODE REVIEW

File: src/features/llm_analytics/components/LLMAnalyticsEmptyState/LLMAnalyticsEmptyState.module.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .stateContainer {
-  margin-bottom: 40px;
+  margin-bottom: 30px;
```
Comment (Natallia-Harshunova) [inline src/features/llm_analytics/components/LLMAnalyticsEmptyState/LLMAnalyticsEmptyState.module.css]: I would do even less than 30px

**Review by Natallia-Harshunova — APPROVED** [review]

File: src/features/llm_analytics/components/LLMAnalyticsPlaceholder/LLMAnalyticsPlaceholder.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          components={{
+            icon1: <SelectArea16 />,
+            icon2: <EditGeometry16 />,
+            icon3: <Upload16 />,
+          }}
+        />
+      </div>
+      <div>
+        <Text type="long-m">{i18n.t('llm_analytics.placeholder.you_can_also')}</Text>
+      </div>
```
[the remark this answers is not in the document]
Comment (Natallia-Harshunova) [reply src/features/llm_analytics/components/LLMAnalyticsPlaceholder/LLMAnalyticsPlaceholder.tsx]: if Viachaslau had reviewed this code, he would certainly have written a comment about that useless div)

[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/llm_analytics/components/LLMAnalyticsPlaceholder/LLMAnalyticsPlaceholder.tsx]: Fixed

File: src/features/llm_analytics/components/LLMAnalyticsEmptyState/LLMAnalyticsEmptyState.module.css (hunk 1 above)
Comment (albaranau) [reply src/features/llm_analytics/components/LLMAnalyticsEmptyState/LLMAnalyticsEmptyState.module.css]: Made it 20px

**Pull request merged**